packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -247,11 +247,19 @@ export interface ISemanticColors {
    * Background of a checked and hovered standard button; e.g. bold/italicize/underline text button in toolbar
    */
   buttonBackgroundCheckedHovered: string;
+  /**
+   * Background of a pressed standard button; i.e. currently being clicked by mouse
+   */
+  buttonBackgroundPressed: string;
 
   /**
    * Border of a standard button
    */
   buttonBorder: string;
+  /**
+   * Border of a disabled standard button
+   */
+  buttonBorderDisabled: string;
```

Comment:
We already have existing disabledText and disabledBackground slots. Can you confirm with designer that we shouldn't re-use those slots for buttons?